### PR TITLE
INT-656 SMS

### PR DESCRIPTION
**Pull request by nurutdinov**

### CODE REVIEW

File: talk/talk-api.yaml [hunk 1]
```diff
@@ -1157,6 +1154,161 @@ paths:
...
+        default:
+          description: Bad request, security violation, or internal server error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResponseError'
+    post:
+      tags:
+        - Messages
+      summary: Send message
```
Comment (BullDog57Rus) [inline talk/talk-api.yaml:1188]:
```suggestion
      summary: Create a message in a chat (if present, or else create a chat also) and send the message
```
maybe something like that

**Review by BullDog57Rus — CHANGES_REQUESTED** [review]

File: talk/talk-api.yaml [hunk 2]
```diff
@@ -1157,6 +1154,161 @@ paths:
...
+        - Messages
+      summary: Retrieve a list of chats.
+      operationId: getChats
+      parameters:
+        - $ref: '#/components/parameters/QueryPageNumber'
+        - $ref: '#/components/parameters/QueryPageSize'
+        - $ref: '#/components/parameters/QuerySort'
+        - name: userNumber
+          in: query
+          required: true
```
[the remark this answers is not in the document]
Comment (nurutdinov) [reply talk/talk-api.yaml:1169]: I think we haven't cases when we need to show to users chats from all their phone numbers.

File: talk/talk-api.yaml [hunk 3]
```diff
@@ -1157,6 +1154,161 @@ paths:
...
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Message'
+        default:
+          description: Bad request, security violation, or internal server error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResponseError'
+  /chats/messages:
```
[the remark this answers is not in the document]
Comment (nurutdinov) [reply talk/talk-api.yaml:1209]: We decided to change id to userNumber and otherPersonNumber for case when we start chat from contact list without chat id.

File: talk/talk-api.yaml [hunk 4]
```diff
@@ -1157,6 +1154,161 @@ paths:
...
+        - name: userNumber
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: otherPersonNumber
+          in: query
+          required: true
+          schema:
+            type: string
```
[the remark this answers is not in the document]
Comment (nurutdinov) [reply talk/talk-api.yaml:1228]: see prev comment

File: talk/talk-api.yaml [hunk 5]
```diff
@@ -4085,3 +4236,158 @@ components:
...
+        - id
+        - userNumber
+        - otherPerson
+        - lastMessage
+        - unreadMessagesCount
+      properties:
+        id:
+          $ref: '#/components/schemas/Id'
+        userNumber:
+          type: string
```
[the remark this answers is not in the document]
Comment (nurutdinov) [reply talk/talk-api.yaml:4264]: I think unuseful only for number and heavy for the long messages list.

**Review by BullDog57Rus — APPROVED** [review]

**Pull request merged**